pandas/core/arrays/integer.py

```diff
@@ -94,7 +94,8 @@ def to_integer_array(values, dtype=None):
     ------
     TypeError if incompatible types
     """
```

Comment:
is there still a ref to to_integer_array? (I see it in the diff, but I also see that you changed it above)